### PR TITLE
Feature/farm 402 playground widget

### PR DESCRIPTION
**Pull request by davidfranquet**
[FARM-402]

#### 📲 What

Create a Playground for widgets to ease the widget testing

#### 🤔 Why
		
We need it because with this we will have an overview of all the widgets and we will can test/see the incoming widgets without having to modify any piece of code, just adding the widget to the widget list that will render the playground
		
#### 🛠 How
		
Created a new Widget to render any incoming Widget with a Search functionality

#### ✅ Acceptance criteria

- [ ] Design Review for UI with BA completed. 
- [x] Manually tested and verified on Android device/emulator (min. Lollipop 5.1)
- [ ] Showcase video / automation test
- [ ] Passing all tests
- [x] Rebased/merged with latest changes from development and re-tested
- [ ] Removed unsed comments. TODOs must have JIRA for future work.

#### Coding Standards Checklist
- [ ] unit tests 80% coverage on testable code (functions, methods, class)
- [x] (Architecture) redux state, reducers, actions, middleware defined under lib/redux
- [x] ui elements defined under lib/ui. 
- [ ] Strings ready for localisation (e.g. defined in lib/utils/strings.dart)
- [ ] Assets (images/icons path) defined in lib/utils/assets.dart
- [ ] Colors defined in lib/utils/colors.dart
- [ ] Dimensions (ie margins padding) defined in lib/utils/dimens.dart
- [ ] TextStyles defined in lib/utils/styles.dart

Recommended Style Guide: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo

#### 🕵️‍♂️ How to test

Notes for QA

#### Reviewers

@farmsmart/amido @farmsmart/wamf

### CODE REVIEW

File: lib/ui/home.dart [hunk 1]
```diff
@@ -25,6 +27,7 @@ class Home extends StatefulWidget {
 
 class _HomeState extends State<Home> with WidgetsBindingObserver {
   HomeViewmodel homeViewModel;
+  static const _debugString = 'Debug';
```
Comment (LeeMatthewHiggins) [inline lib/ui/home.dart]: pop that next to the other tabs string for now.

File: lib/ui/playground/playground_view.dart [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+    setState(() {
+      items.clear();
+      items.addAll(widget.widgetList);
+    });
+  }
+
+  ListView _listView() {
+    return ListView.builder(
+        shrinkWrap: true,
+        itemCount: items.length + _hardcodedItemSize,
```
Comment (LeeMatthewHiggins) [inline lib/ui/playground/playground_view.dart]: "size" makes me thing of a UI size, I wonder if that should be called _hardcodedItemCount

File: lib/ui/playground/playground_view.dart [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+      },
+      controller: editingController,
+      decoration: InputDecoration(
+        labelText: _searchText,
+        hintText: _searchText,
+        labelStyle: TextStyle(color: widget.appBarColor),
+        prefixIcon: Icon(Icons.search, color: widget.appBarColor),
+        border: OutlineInputBorder(
+            borderSide: BorderSide(color: Colors.red),
+            borderRadius: BorderRadius.all(Radius.circular(10.0))),
```
Comment (LeeMatthewHiggins) [inline lib/ui/playground/playground_view.dart]: magic number

**Review by LeeMatthewHiggins — DISMISSED** [review]
minor naming comments.
Looks great, lets get this in!

**Comment by raubreysmith** [comment]
how does this work? Is it always displayed?

**Review by LeeMatthewHiggins — APPROVED** [review]

**Comment by LeeMatthewHiggins** [comment]
> how does this work? Is it always displayed?

It will be displayed while in development, its the last tab. We will add the widgets as they are made in other PR´s

**Review by LeeMatthewHiggins — APPROVED** [review]

**Review by jessicahernandezhv — APPROVED** [review]

**Pull request merged**